docs/en/ingest-management/release-notes/release-notes-7.13.asciidoc

```diff
@@ -4,15 +4,39 @@
 :agent-issue: https://github.com/elastic/beats/issues/
 :agent-pull: https://github.com/elastic/beats/pull/
 
-//QUESTION: Any other repos that are needed here? ^^
+[[release-notes]]
+= Release notes
+
+This section summarizes the changes in each release.
+
+* <<release-notes-7.13.1>>
+
+* <<release-notes-7.13.0>>
+
+Also see:
+
+* {kibana-ref}/release-notes.html[{kib} release notes]
+* {beats-ref}/release-notes.html[{beats} release notes]
+
+[[release-notes-7.13.1]]
+== {fleet} and {agent} 7.13.1
+
+The 7.13.1 release includes the following bug fixes.
+
+[discrete]
+[[bug-fixes-7.13.1]]
+=== Bug fixes
+
+{fleet}::
+* Add missing install button for integrations that aren't installed yet {kibana-pull}100370[#100370]
+
+//{agent}::
```

Comment:
* Fix unenroll of agents when connected to a on-prem Fleet Server

https://github.com/elastic/fleet-server/pull/381